k8s/infrastructure/webapi.yaml

```diff
@@ -18,6 +18,22 @@ image:
 deploy:
   containerPort: 8080
 
+storage:
+  extraVolumes:
+    - name: elastic-certs
+      secret:
+        secretName: elastic-certificates
+    
+  extraVolumeMounts: 
+    - name: elastic-certs
+      mountPath: "/workspace/elastic/certs"
+      readOnly: true 
+
+initContainers:
+  - name: init-myservice
+    image: busybox:1.28
+    command: ['sh', '-c', "mkdir -p /workspace/ca-certificates/binding/; cat /workspace/elastic/certs/ca.crt /workspace/elastic/certs/tls.crt /workspace/elastic/certs/tls.key > /workspace/ca-certificates/binding/outschool-ca.pem"]
```

Comment:
maybe we can enable this thing? [https://cert-manager.io/docs/usage/certificate/#additional-certificate-output-formats](https://cert-manager.io/docs/usage/certificate/#additional-certificate-output-formats)